lua/lv-which-key/init.lua

```diff
@@ -138,28 +105,16 @@ local mappings = {
     },
     l = {
         name = "LSP",
-        a = {"<cmd>Lspsaga code_action<cr>", "Code Action"},
```

Comment:
You removed entries here, but then completely duplicated the entry below, this needs to be reworked.

Reply:
I merged the new keys with the already defined ones !